content/en/apps/tutorials/local-setup.md

```diff
@@ -67,12 +67,15 @@ COUCHDB_PASSWORD=password
 EOF
 ```
 
-Run the following command to start your CHT instance using Docker Compose:
+Run the following command to start your CHT instance using Docker Compose. Ensure that Docker is installed and running:
 
 ```shell
 cd ~/cht-local-setup/upgrade/
 docker-compose up
 ```
+Note that the first time you run your CHT instance it may take a while. Incase you run into issues running your docker file, ensure that the following setting in Docker is checked.
```

Comment:
I am not sure if this step is mandatory. I know that Mac M1 users ran into an error (do you remember what the error was @jonathanvq ?) and had to check the box, but I am not sure if it the case for every user.

Reply:
Should be fine for everyone to follow!  macOS and Windows folks should check it, and not that many LInux folks are running Docker Desktop for LInux, but it should be OK for them too.